src/modules/user/user.usecase.ts

```diff
@@ -183,6 +192,10 @@ export class UserUseCases {
     const userPass = this.cryptoService.decryptText(password);
     const userSalt = this.cryptoService.decryptText(salt);
 
+    if (userPass.length > MAX_PASSWORD_LENGTH) {
+      throw new PasswordExceededLimitLengthError();
```

Comment:
I believe you are validating the hashed password not the submitted 🤔 

Correct me if i'm wrong though